userspace/engine/falco_engine.h

```diff
@@ -196,6 +221,7 @@ class falco_engine
 		std::string format;
 		std::set<std::string> exception_fields;
 		std::set<std::string> tags;
+		std::unordered_map<std::string, std::pair<std::string, bool>> extra_output_fields;
```

Comment:
I saw this is written many times, can we `typedef` it to a meaningful name?
Like:
```
typedef extra_output_field_t std::unordered_map<std::string, std::pair<std::string, bool>>
```

Reply:
Thanks! Done.